frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -521,14 +71,17 @@ export const VmActionFactory = {
 
     return {
       id: 'vm-action-start',
-      disabled: vmStatusBundle?.status?.isMigrating() || isVMRunningOrExpectedRunning(vm, vmi),
-      label: i18next.t('kubevirt-plugin~Start Virtual Machine'),
+      disabled:
+        vmStatusBundle?.status?.isMigrating() ||
+        isVMRunningOrExpectedRunning(vm, vmi) ||
+        isVMIPaused(vmi),
+      label: i18next.t('kubevirt-plugin~Start'),
       cta: () => {
         if (!vmStatusBundle?.status?.isImporting()) {
           startVM(vm);
         } else {
           confirmModal({
-            title: i18next.t('kubevirt-plugin~ Start Virtual Machine'),
+            title: i18next.t('kubevirt-plugin~ Start'),
```

Comment:
why do we have a space before Start ?
do we need to play with css ? confirm model does not print the title correclty ?